modules/adnuntiusRtdProvider.js

```diff
@@ -52,12 +51,12 @@ function setGlobalConfig(config, segments) {
     }
   }
   if (config.params && config.params.bidders) {
-    pbjsG.setBidderConfig(mergeBidderConfig({
+    pbjsG.setBidderConfig(mergeDeep(pbjsG, {
```

Comment:
This would merge the entire pbjs global object with ortb2 data...? This is definitely not what we want here. Was there an issue using mergeConfig/mergeBidderConfig? We want only the corresponding configs to be merged here.

Reply:
I was not 100% sure on how to use it and this one seemt do be ok do you have an example on how I would use it?

Reply:
You would just need to pass in the data to each of the corresponding functions. They will handle retrieving the existing data, merging the two, and setting with the newly merged data. Not sure if there are docs or not on it, but it should be treated similarly to setConfig/setBidderConfig. I am seeing now the first time you implemented it was slightly incorrect because you were still calling setConfig/setBidderConfig. You should be able to just run as follows:

```
const pbjsG = getGlobal()
  const ortbSegments = {
    ortb2: {
      user: {
        data: [{
          name: 'adnuntius',
          segment: segments
        }]
      }
    }
  }
...
if (...) {
    pbjsG.mergeBidderConfig({
        bidders: config.params.bidders,
        config: ortbSegments
    })
} else {
    pbjsG.mergeConfig(ortbSegments)
}
```

Reply:
That made more sense once I understood it, thank you! :D